modules/primer-marketing-utilities/docs/layout.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Layout
+status: New release
+status_issue: https://github.com/github/design-systems/issues/325
+---
+
+The following layout utilities are meant to used in addition to those within primer-core.
```

Comment:
The first paragraph should describe what these styles are, this para will get used in other contexts such as og metadata in future. Also, it's nice to hyperlink things that are documented in the style. Suggest something like this:

> Marketing layout utilities build on top of [primer-core utilities](layout), adding the option of responsive positioning.